src/metaprob/builtin_impl.clj

```diff
@@ -15,7 +15,7 @@
 (declare procedure-name)
 
 (defn addr [& keys]
-  (if (= keys nil)
+  (if (nil? keys)
```

Comment:
I don't understand why this is better. When I wrote (= keys nil) I didn't have to learn a new function. Now in order to read this code I have to look up the meaning of `nil?`.  It may be idiomatic, but generally idioms are good for the in-crowd and clojure experts, and terrible for people like me.

Reply:
Oh I think I figured it out: `=` is expensive, `(nil? x)` is going to cut out a method call.  I like that.  So maybe keep it.